yt/yt/server/master/chunk_server/chunk_owner_node_proxy.cpp

```diff
@@ -1135,6 +1135,7 @@ bool TChunkOwnerNodeProxy::SetBuiltinAttribute(
             }
 
             ValidatePermission(EPermissionCheckScope::This, EPermission::Write);
+            ValidateErasureCodec(value, config->ForbiddenErasureCodecIds);
```

Comment:
This PR changes mutation semantics so reign should be promoted. See https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/server/master/cell_master/serialize.h